forgerock-auth/src/main/java/org/forgerock/android/auth/callback/DeviceBindingCallback.kt

```diff
@@ -245,6 +245,7 @@ open class DeviceBindingCallback : AbstractCallback, Binding {
                             deviceBindingRepository.persist(it)
                             val jws = deviceAuthenticator.sign(context,
                                 kp,
+                                status.signature,
```

Comment:
An unlocked signature is added to the interface. We need to use the same instance from CryptoObject to the Signing.